cpp/src/slice2matlab/Main.cpp

```diff
@@ -314,7 +50,20 @@ namespace
 
     void openClass(const string& abs, const string& dir, IceInternal::Output& out)
     {
-        vector<string> v = splitAbsoluteName(abs);
+        // Split the absolute name into individual components.
```

Comment:
We have a quite similar method in slice2java, maybe we can refactor this code and add helper in Slice/Util.h, anyway that would be for another PR.